src/main/resources/templates/projects/list.html

```diff
@@ -1,67 +1,94 @@
 <!DOCTYPE html>
 <html lang="en" data-th-replace="~{base :: layout(~{::title}, ~{::main})}">
-<head>
-  <title>Recurse Center Community Portfolio</title>
-  <link rel="stylesheet" href="../../static/style.css"/>
-</head>
-<body>
-<main>
-  <h2>Project List</h2>
-  <section>
-    <ul
-      class="global-project-list"
-      data-th-fragment="list(projects)"
-    >
-      <li
-        class="global-project-list-item"
-        data-th-each="project : ${projects}"
-      >
-        <a
-          href="projects/public.html"
-          data-th-href="@{/project/{id}(id=${project.projectId})}"
-          data-th-text="${project.name}"
-        >Project Name</a>,
-        by
-        <span
-          class="global-project-list-item-author"
-          data-th-each="author : ${project.authors}"
-        >
-          <a
-            href="users/public.html"
-            data-th-href="@{/user/{id}(id=${author.userId})}"
-            data-th-text="${author.name}"
-          >Author Name</a><span
-            data-th-if="not ${authorStat.last}"
-          >, </span>
-        </span>
-        <ul class="global-project-list-item-tag-list">
+  <head>
+    <title>Recurse Center Community Portfolio</title>
+    <link rel="stylesheet" href="/style.css" />
```

Comment:
Did you find that you needed to change this for the stylesheet to load correctly when running the application?

The way it's supposed to work is that the contents of the `head` element - including this `link` element - are [replaced by ThymeLeaf](https://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html#template-layout) when rendered because of the `data-th-replace` directive on line 2. The `link` with the relative URL is still useful in development, though, because it means you can load the template in your browser and it will load the stylesheet correctly!

If the application loads the stylesheet correctly without this change, would you mind reverting it, please?